url-plugins/checkstyle.xml

```diff
@@ -0,0 +1,400 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You shouldn't build directly in this directory. This repo is a multi-module project and it should always run from the top level. The checkstyle plugin should be configured at top level pom as well. See how cdap do it.

To build module individually, you should run `mvn clean package -pl url-plugins -am` at the top level.